catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchSource.java

```diff
@@ -765,6 +771,14 @@ public void setConfigurationPid(String configurationPid) {
         this.configurationPid = configurationPid;
     }
 
+    public List<String> getMarkUpList() {
+        return markUpList;
+    }
+
+    public void setMarkUpList(List<String> markUpList) {
+        this.markUpList = markUpList;
```

Comment:
We could turn the list into a set here, if these are expected to be unique, then use a Set lookup to simplify the below code.